index.html

```diff
@@ -1,111 +1,173 @@
-                  class="sns-btn"
-                >
-                  twitter <!-- Link to social media 2 -->
-                </a>
-              </div>
-
-              <div class="portfolio-btn-box">
-                <p>
-                  Do you want to see error? <!-- Any text -->
-                </p>
-                <div class="portfolio">
-                  <button id="error-btn" class="error-btn">
-                    error?
-                  </button>
-                  <a
-                    href="https://ayakoyokoya.github.io/yaba-kawaii-audio-visualizer/dist/"
-                    target="_blank"
-                    class="portfolio-btn"
-                  >
-                    yes! 
-                  </a>
-                </div>
-              </div>
-            </div>
-          </div>
+  <header class="header">
+    <img src="img/Logos.svg" alt="" class="logo">
+    <nav class="navigation">
+      <ul class="navigation-list">
+        <li class="navigation item"><a href="" class="nav-link">Home</a></li>
+        <li class="navigation item"><a href="" class="nav-link">Blog</a></li>
+        <li class="navigation-item"><a href="" class="nav-link">Catalog</a></li>
+        <li class="navigation-item"><a href="" class="nav-link">Contact Us</a></li>
+      </ul>
+    </nav>
+    <button class="search-bar">Search</button>
+    <a href="" class="discord">Discord</a>
+    <button class="connect">Connect</button>
+  </header>
+  <main class="content">
+    <section class="headline">
+      <h1>With <strong>Binabox</strong> Explore NFT collection</h1>
+      <p>We are the best way to check the rarity of NFT collection</p>
+      <ul class="achievements">
+       <li class="achievement-info">2240+</li>
+       <li class="achievement-info">Total item</li>
+       <li class="achievement-info">1000+</li>
+       <li class="achievement-info">Profile whitelisted</li>
+      </ul>
+    </section>
+    <section class="catalog">
+      <h2>Catalog</h2>
+      <a href="">See all</a>
+      <button>Right</button>
+      <button>Left</button>
+      <div class="catalog-item">
+        <img src="" alt="" class="catalog-image">
+        <p class="catalog-price">$199</p>
+        <p class="catalog-describtion">Name product</p>
+      </div>
+      <div class="catalog-item">
+        <img src="" alt="" class="catalog-image">
+        <p class="catalog-price">$199</p>
+        <p class="catalog-describtion">Name Product</p>
+      </div>
+      <div class="catalog-item">
+        <img src="" alt="" class="catalog-image">
+        <p class="catalog-price">$199</p>
+        <p class="catalog-describtion">Name Product</p>
+      </div>
+      <div class="catalog-item">
+        <img src="" alt="" class="catalog-image">
+        <p class="catalog-price">$199</p>
+        <p class="catalog-describtion">Name Product</p>
+      </div>
+    </section>
+    <section class="our-blog">
+      <h2>Our blog</h2>
+      <article class="sonic-live">
```

Comment:
@Alinchik1 Great usage of "article" tag!

Reply:
Thank you